### ISSUE TITLE
[IDL] Add Animation pages and Illustration section Usage tabs pages  

### ISSUE THREAD

**Issue by mjabbink** [carbon-design-system/gatsby-theme-carbon#825]
Lori Mapes to review and edit content first.

 - [ ]  We have a new section to add in the left nav called “Animation”

Animation                  
     Overview
     Classic principles
     Tips & techniques

- [ ]  We have “usage” tab pages one for each illustration style (Line, Flat and Isometric). Three pages total.
          

- [ ]  We have content to add to the “Production” tab in Isometric style

Figma:
https://www.figma.com/file/x3607TQjOoyA1BV1a5lhLm/IBM_M4I_IDLmockup_a08?node-id=0%3A1

**Comment by sadekbazaraa** [comment]
Cool, thanks @mjabbink. Are we putting all IDL issues in this repo now? 

Lori shared comments from Peggy and herself with me on Friday. Here's the file. I haven't had a chance to review it yet.
https://ibm.box.com/s/6io9gcwya1sjntmw3w7b2ntsni4p9eme

**Comment by LMapes** [comment]
@sadekbazaraa @mjabbink I can comment

**Comment by sadekbazaraa** [comment]
Great : ) @LMapes @mjabbink I've been going through Lori and Peggy's doc posted above and have integrated most of the comments and changes into a new doc where I'm tracking any additional changes or suggestions. This way will have a master manuscript to refer back to. I can pass this doc back to Lori for further review when ready.

Meanwhile, I've started coding pages. It's quite a cumbersome process which I can explain further on our next standup but it's moving along.

**Comment by LMapes** [comment]
@mjabbink @sadekbazaraa so, at our last Monday standup, Peggy and I admitted we were stumped to discover that there is a new (to us) effort for designers to code directly in Markdown. Did we miss a training on Markdown? We've only had the briefest taste in that first meeting with Connor back in February. We'd like to be up to speed too, so don't forget to bring us along.

**Comment by mjabbink** [comment]
Yes that is the goal so our team can tackle copy edits and link updates as well as getting a templates page up in staging from designers. We want devs to focus on dev work.

**Comment by mjabbink** [comment]
The check boxes above under Development header are for dev for example
